### PR TITLE
Last minute changes to data library

### PR DESCRIPTION
**Pull request by Kucharssim**
- replaces the previous gigantic prophet jasp file (related to https://github.com/jasp-stats/jaspProphet/pull/43)
- adds two SEM datasets that will be used in upcoming blog posts (there are no  analysese in the .jasp files yet though) - on request by @AlexanderLyNL and @JMBKoch

### CODE REVIEW

File: Resources/Data Sets/index_nl.json [hunk 1]
```diff
@@ -779,6 +779,20 @@
           "description": "Indicatoren van industrialisatie in 1960, en politieke democratie in 1960 en 1965. <br><br> Het voorbeeldbestand in JASP laat zien hoe structural equation modeling kan worden toegepast. <br><br> <i>Dataset in R.<\/i>",
           "kind": "file",
           "associated_datafile": "Political Democracy.csv"
+        },
+        {
+          "name": "Grand Point Average",
+          "path": "Grand Point Average.jasp",
+          "description": "Grand Point Average of 200 college students across 6 semesters, along with information on gender, high school GPA and whether they were admitted to their university of choice. <br><br> The JASP file currently does not contain any analysis. <br><br> <i>Data from Hox, Moerbeek and Van de Schoot (2017).<\/i>",
```
Comment (vandenman) [inline Resources/Data Sets/index_nl.json]:
```suggestion
          "description": "Gemiddeld aantal studiepunten van 200 studenten over 6 semesters, samen met informatie over geslacht, aantal studiepunten op de middelbare school en of ze zijn toegelaten tot de universiteit van hun keuze. <br><br>Het JASP-bestand bevat momenteel geen analyses. <br><br> <i>Dataset van Hox, Moerbeek, en Van de Schoot (2017).<\/i>",
```
I also added an Oxford comma.

File: Resources/Data Sets/index.json [hunk 2]
```diff
@@ -779,6 +779,20 @@
           "description": "Indicators of industralization in 1960, and political democracy in 1960 and 1965. <br><br> The example JASP file demonstrates the use of structural equation modeling.<br><br> <i>Data set included in R.<\/i>",
           "kind": "file",
           "associated_datafile": "Political Democracy.csv"
+        },
+        {
+          "name": "Grand Point Average",
+          "path": "Grand Point Average.jasp",
+          "description": "Grand Point Average of 200 college students across 6 semesters, along with information on gender, high school GPA and whether they were admitted to their university of choice. <br><br> The JASP file currently does not contain any analysis. <br><br> <i>Data from Hox, Moerbeek and Van de Schoot (2017).<\/i>",
```
Comment (vandenman) [inline Resources/Data Sets/index.json]: Should there be an Oxford comma here?
```
... Moerbeek, and Van de Schoot ...
            ^
```

File: Resources/Data Sets/index_nl.json [hunk 3]
```diff
@@ -779,6 +779,20 @@
...
+          "kind": "file",
+          "associated_datafile": "Grand Point Average.csv"
+        },
+        {
+          "name": "Social Participation",
+          "path": "Social Participation.jasp",
+          "description": "Measurements of Social Participation (church membership, memberships in voluntary organizations, and number of friends seen), as well as measures on potential causes of social interaction (income, occupation and education). <br><br> The JASP file currently does not contain any analysis. <br><br>  <i>Simulated data based on Hodge and Treiman (1986).<\/i>",
```
Comment (vandenman) [inline Resources/Data Sets/index_nl.json]:
```suggestion
          "description": "Metingen van sociale participatie (kerklidmaatschap, lidmaatschappen van vrijwilligersorganisaties, en aantal vrienden dat wordt gezien), alsmede metingen van mogelijke oorzaken van sociale interactie (inkomen, beroep en opleiding). <br><br> Het JASP-bestand bevat momenteel geen analyses. <br><br> <i>Gesimuleerde gegevens gebaseerd op Hodge en Treiman (1986).<\/i>",
```

**Review by vandenman — APPROVED** [review]
Looks good to me! 

I deepl translated the descriptions to Dutch. We could also do that for German I guess.

**Comment by boutinb** [comment]
@Kucharssim If you add the changes proposed by Don, you can merge it.

File: Resources/Data Sets/index.json (hunk 2 above)
Comment (Kucharssim) [reply Resources/Data Sets/index.json]: yes, there should, thanks!

**Pull request merged**